e2e-polybft/cardanofw/test-apex-system.go

```diff
@@ -259,6 +315,35 @@ func (a *ApexSystem) FundWallets(ctx context.Context) error {
 	})
 }
 
+func (a *ApexSystem) FundWalletsSkyline(ctx context.Context) error {
```

Comment:
This method should not exists
if there are tokens in config than tokens should be funded too